schema2.sql

```diff
@@ -0,0 +1,14 @@
+-- Harry Potter Quiz 
+
+-- Questions Table
+CREATE TABLE questions(
+    question text,
+    difficulty integer
+);
+
+-- Answers Table
+CREATE TABLE answers(
+    answer text,
+    correct boolean,
+    time_lapsed integer
```

Comment:
Oooh! Interesting addition! I can see this being an important part of a learning app.